Examples/Makefile.in

```
@@ -1381,14 +1381,15 @@ r_clean:
 SCILAB = @SCILAB@
 SCILAB_INC= @SCILABINCLUDE@
 SCILAB_OPT = @SCILABOPT@
+SCILAB_VERSION = @SCILAB_FULL_VERSION@
```

`-targetversion` should take the 'major version', so suggest:
```suggestion
SCILAB_MAJOR_VERSION = @SCILAB_MAJOR_VERSION@
```
plus other necessary changes.